src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -84,47 +86,30 @@ public ProcessStats GetStats()
 
         private void OnErrorDataReceived(object sender, DataReceivedEventArgs e)
         {
-            // TODO: per language stdout/err parser?
             if (e.Data != null)
             {
                 string msg = e.Data;
-                if (msg.IndexOf("warn", StringComparison.OrdinalIgnoreCase) > -1)
+                if (_useStdErrStreamForErrorsOnly)
                 {
```

Comment:
In Node.js worker, we add `[warn]` and `[error]` so the "warn" case would see a change in behavior. Can we make this so that we only look at the "_useStdErrStreamForErrorsOnly" property in cases where we're not sure what to do with the message(i.e. change the "else" from information => error?)

Reply:
Thanks! Simplified and updated!